apps/web/src/app/api/projects/[projectId]/commits/[commitUuid]/documents/[documentUuid]/evaluation-results-by-document-content/[evaluationId]/pagination/route.ts

```diff
@@ -0,0 +1,81 @@
+import { Workspace } from '@latitude-data/core/browser'
+import { buildPagination } from '@latitude-data/core/lib/pagination/buildPagination'
+import {
+  CommitsRepository,
+  EvaluationsRepository,
+} from '@latitude-data/core/repositories'
+import { computeEvaluationResultsWithMetadataCount } from '@latitude-data/core/services/evaluationResults/computeEvaluationResultsWithMetadata'
+import { authHandler } from '$/middlewares/authHandler'
+import { errorHandler } from '$/middlewares/errorHandler'
+import { ROUTES } from '$/services/routes'
+import { NextRequest, NextResponse } from 'next/server'
+
+function pageUrl(params: {
+  projectId: string
+  commitUuid: string
+  documentUuid: string
+  evaluationId: string
+}) {
+  return ROUTES.projects
+    .detail({ id: Number(params.projectId) })
+    .commits.detail({ uuid: params.commitUuid })
+    .documents.detail({ uuid: params.documentUuid })
+    .evaluations.detail(Number(params.evaluationId)).root
+}
+
+export const GET = errorHandler(
+  authHandler(
+    async (
+      req: NextRequest,
+      {
+        params,
+        workspace,
+      }: {
+        params: {
+          evaluationId: number
+          documentUuid: string
+          commitUuid: string
+          projectId: number
+        }
+        workspace: Workspace
+      },
+    ) => {
+      const searchParams = req.nextUrl.searchParams
+      const evaluationsScope = new EvaluationsRepository(workspace.id)
+      const evaluation = await evaluationsScope
+        .find(params.evaluationId)
+        .then((r) => r.unwrap())
+      const commitsScope = new CommitsRepository(workspace.id)
+      const commit = await commitsScope
+        .getCommitByUuid({
+          uuid: params.commitUuid,
+          projectId: params.projectId,
+        })
+        .then((r) => r.unwrap())
+      const countResult = await computeEvaluationResultsWithMetadataCount({
+        workspaceId: workspace.id,
+        evaluation,
+        documentUuid: params.documentUuid,
+        draft: commit,
+      })
```

Comment:
Why is it `withMetadata` if the metadata is not used on the count?

Reply:
Copy & paste. Will change